pkg/proxy/ipvs/proxier.go

```diff
@@ -116,9 +116,9 @@ var iptablesChains = []struct {
 	{utiliptables.TableNAT, KubeFireWallChain},
 	{utiliptables.TableNAT, KubeNodePortChain},
 	{utiliptables.TableNAT, KubeLoadBalancerChain},
+	{utiliptables.TableFilter, KubeForwardChain},
 	{utiliptables.TableNAT, KubeMarkMasqChain},
 	{utiliptables.TableNAT, KubeMarkDropChain},
-	{utiliptables.TableFilter, KubeForwardChain},
```

Comment:
What's the reason for changing the order here?

Reply:
No reason, actually. 😂